sway-lsp/tests/lib.rs

```diff
@@ -1239,6 +1239,7 @@ fn go_to_definition_for_functions() {
 }
 
 #[test]
+#[ignore]
```

Comment:
Can we link this issue here https://github.com/FuelLabs/sway/issues/7025

Reply:
Done.